src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,184 @@
+            IconButtonProps={{ color: 'inherit', size: 'small' }}
+            aria-controls={title}
+            id={title}
+          >
+            <img className={classes.icon} src={icon} alt={title} />
+            <Typography variant="body2">{title}</Typography>
+          </AccordionSummary>
+          <AccordionDetails className={classes.accordionDetail}>
+            <Grid container direction="column">
+              {layersCategories.map(
+                ({ title: categoryTitle, layers, tables }) => (
+                  <Grid
+                    item
+                    key={categoryTitle}
+                    className={classes.categoryContainer}
+                  >
+                    <Typography
+                      variant="body2"
+                      className={classes.categoryTitle}
+                    >
+                      {categoryTitle}
+                    </Typography>
+                    <hr />
+
+                    {layers.map(layer => {
+                      const { id: layerId, title: layerTitle } = layer;
+                      const selected = Boolean(
+                        selectedLayers.find(
+                          ({ id: testId }) => testId === layerId,
+                        ),
+                      );
+                      return (
+                        <div key={layerId} className={classes.layersContainer}>
+                          <Switch
+                            size="small"
+                            color="default"
+                            checked={selected}
+                            onChange={() => toggleLayerValue(selected, layer)}
+                            inputProps={{ 'aria-label': layerTitle }}
+                          />{' '}
+                          <Typography variant="body1">{layerTitle}</Typography>
+                        </div>
+                      );
+                    })}
+
+                    {tables.map(table => (
+                      <Button
+                        id={table.title}
+                        key={table.title}
+                        onClick={() => showTableClicked(table)}
+                      >
+                        <Typography variant="body1">{table.title}</Typography>
+                      </Button>
+                    ))}
+                  </Grid>
+                ),
+              )}
+            </Grid>
+          </AccordionDetails>
+        </Accordion>
+      ))}
+    </>
+  );
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    icon: {
+      width: 18,
+      marginRight: 6,
+    },
+
+    categoryContainer: {
+      marginBottom: 16,
+      '&:last-child': {
+        marginBottom: 0,
+      },
+    },
+
+    categoryTitle: {
+      fontWeight: 'bold',
+      textAlign: 'left',
+    },
+
+    layersContainer: {
+      display: 'flex',
+      marginBottom: 8,
+    },
+
+    accordion: {
+      boxShadow: 'none',
+      '&.Mui-expanded': {
+        margin: 0,
+        '&:before': {
```

Comment:
Not sure what this is selecting exactly. A comment to explain why this is here?

Reply:
So I want to custom the accordion a little bit because I don't really like how the default element added padding in the top and bottom when the accordion is expanded. the border also disappear when it expanded so I added 2 custom styles here:
`margin: 0` to remove the padding, and `opacity: 1` to keep the border

![image](https://user-images.githubusercontent.com/25983260/111446304-d703ba80-873e-11eb-9b0d-ff0a50a1ff2f.png)

It just my preference but it also fine to remove the custom if we want to use the default one. What do you think?

Reply:
All good, we just need a comment explaining this in code. Also, you should move this to the global theme as indicated here:
https://github.com/WFP-VAM/prism-frontend/pull/129#discussion_r596799486